server/mem_agent/tools.py

```
@@ -0,0 +1,359 @@
+        file_or_dir_path: The path to the file or directory. 
+                          If empty string, returns total memory directory size.
+
+    Returns:
+        The size of the file or directory in bytes.
+    """
+    # Handle empty string by returning total memory size
+    if not file_or_dir_path or file_or_dir_path == "":
+        # Get the current working directory (which should be the memory root)
+        cwd = os.getcwd()
+        total_size = 0
+        for dirpath, dirnames, filenames in os.walk(cwd):
```

Ruff (0.14.1)

Loop control variable `dirnames` not used within loop body

Rename unused `dirnames` to `_dirnames`

(B007)